### PR TITLE
feat: add remote support for vscode project manager extension

### PR DESCRIPTION
**Pull request by TobiasYin**
## Description
The original extension of *Visual Studio Code - Project Manager* does not support ssh remote project. The pr add remote support for the extension. 

## Checklist

- [ ] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [ ] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [ ] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [ ] I checked that files in the `assets` folder are used by the extension itself
- [ ] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
@MarkusLanger can you check this?

File: extensions/visual-studio-code-project-manager/src/search-project-manager-projects.tsx [hunk 1]
```diff
@@ -255,3 +269,13 @@ function isGitRepo(path: string): boolean {
   const gitConfig = config.sync({ cwd: path, path: ".git/config", expandKeys: true });
   return !!gitConfig.core;
 }
+
+function isRemotePreject(path: string): boolean {
```
Comment (MarkusLanger) [inline extensions/visual-studio-code-project-manager/src/search-project-manager-projects.tsx:273]: @TobiasYin Please fix typo "isRemoteProject"

Comment (TobiasYin) [reply extensions/visual-studio-code-project-manager/src/search-project-manager-projects.tsx]: Thanks, I don't notice this. Already fixed.

**Comment by MarkusLanger** [comment]
@pernielsentikaer Looks good to me. I haven't tested the code but I took a look at it.

**Comment by MarkusLanger** [comment]
@TobiasYin You can also update the contributors
https://github.com/raycast/extensions/blob/main/extensions/visual-studio-code-project-manager/package.json#L8

**Review by pernielsentikaer — APPROVED** [review]

**Pull request merged**